hazelcast/src/main/java/com/hazelcast/spi/impl/operationservice/impl/OperationServiceImpl.java

```diff
@@ -148,22 +145,17 @@ public OperationServiceImpl(NodeEngineImpl nodeEngine) {
         this.nodeEngine = nodeEngine;
         this.node = nodeEngine.getNode();
         this.thisAddress = node.getThisAddress();
-        this.logger = node.getLogger(OperationService.class);
+        this.logger = node.loggingService.getLogger(OperationService.class);
```

Comment:
I believe it should be changed. Node isn't responsible for providing an API to create a logger; that is already the concern of the LoggingService.